.cbmc-batch/include/make_common_data_structures.h

```diff
@@ -35,6 +35,7 @@ void ensure_sig_ctx_has_allocated_members(struct aws_cryptosdk_sig_ctx *ctx);
 bool aws_cryptosdk_edk_list_is_bounded(
     const struct aws_array_list *const list, const size_t max_initial_item_allocation);
 bool aws_cryptosdk_edk_list_elements_are_bounded(const struct aws_array_list *const list, const size_t max_item_size);
+bool ensure_cryptosdk_edk_list_elements_are_readable(const struct aws_array_list *const list);
```

Comment:
`ensure_nodet...`?

Reply:
Yeah that would be a better name! I've since discovered the aws_cryptosdk_edk_list_elements_are_valid function which ensures the readability condition I need so I've switched to using that instead and I've removed this function.